src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -47,11 +51,18 @@ private void processArg(Instrumenter jacoco, String arg) throws Exception {
         Path inPath = Paths.get(parts[0]);
         Path outPath = Paths.get(parts[1]);
         String srcs = parts[2];
+
+        // Use a directory for coverage metadata  that is unique to each built jar. Avoids
```

Comment:
tiny tiny tiny nit: double space between `metdata` and `that`

Reply:
Good catch :) I think I copy-pasted think bit from rules_java... ;)